src/components/sdm/ListItem/ListItemCollapsible.js

```diff
@@ -33,3 +33,28 @@ export const ListItemCollapsible = sources => {
     DOM,
   }
 }
+
+export const ListItemCollapsibleDumb = sources => {
```

Comment:
Can you put a small comment above here to say what the difference is between Smart and Dumb components, as it's not so obvious from the code.